LIBVmonitor0.11/LIBVmonitor0.11.ino

```diff
@@ -162,31 +194,27 @@ void loop() {
 }
```

Comment:
If it is practical, move the LCD updating code (i.e. everything from line 174-193) to a helper function called updateLCD() or similar.